cli/ballerina-cli/src/main/java/io/ballerina/cli/task/RunTestsTask.java

```diff
@@ -216,52 +201,60 @@ public void execute(Project project) {
                 suite.setTests(TesterinaUtils.getSingleExecutionTests(suite, singleExecTests));
             }
             suite.setReportRequired(report || coverage);
-            Collection<Path> dependencies = jarResolver.getJarFilePathsRequiredForTestExecution(moduleName);
-            if (project.kind() == ProjectKind.SINGLE_FILE_PROJECT) {
-                out.println("\t" + module.document(module.documentIds().iterator().next()).name());
-            } else {
-                out.println("\t" + module.moduleName().toString());
-            }
-            writeToJson(suite, moduleTestCachePath);
-            int testResult;
+
```

Comment:
Let's get rid of unnecessary new lines

Reply:
Removed instances of unnecessary new lines.